packages/svelte/src/compiler/phases/constants.js

```diff
@@ -149,6 +149,39 @@ export const SVGElements = [
 	'vkern'
 ];
 
+export const MathMLElements = [
```

Comment:
Taken from https://www.w3.org/TR/mathml-core/#mathml-elements-and-attributes

MathML Core is the standard that is reaching interopable support in browsers.